config/ember-try.js

```diff
@@ -22,25 +22,6 @@ module.exports = function() {
             }
           }
         },
-        {
-          name: 'ember-2.0',
```

Comment:
Do we want this removed?

Reply:
my feeling has been that there isn't a lot of benefit to testing a release this old since there _shouldn't_ be much reason users would be stuck on 2.0 vs one of the later 2.x series releases.